### PR TITLE
Add target_hint to BrowserOptions

### PR DESCRIPTION
**Pull request by Azorlogh**
See #44

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -162,6 +163,7 @@ impl std::default::Default for BrowserOptions {
     fn default() -> Self {
         BrowserOptions {
             suppress_output: true,
+            target_hint: String::from("_blank"),
```
Comment (amodm) [inline src/lib.rs:166]: Should be `option_env!("WEBBROWSER_WASM_TARGET")` to ensure back comptability.

**Review by amodm — CHANGES_REQUESTED** [review]
Given the test failures, I think it's best to move `BrowserOptions` to a [non-consuming builder pattern](https://doc.rust-lang.org/1.0.0/style/ownership/builders.html#non-consuming-builders-(preferred):). This way devs can selectively override the specific options they want to override

e.g. instead of saying `BrowserOptions { ... }`, it should be something like `BrowserOptions::new().with_target_hint("_self")`. If you're unsure about it, [allow me to make changes to this PR](https://tighten.com/blog/adding-commits-to-a-pull-request/) so that I can get the tests to pass.

File: src/lib.rs (hunk 1 above)
Comment (Azorlogh) [reply src/lib.rs]: Oh right, fixed.

**Comment by Azorlogh** [comment]
I changed it to the non consuming builder pattern & fixed the rustdoc. Feel free to make edits if necessary

**Review by amodm — APPROVED** [review]

**Comment by amodm** [comment]
Thanks for your first contribution here @Azorlogh. This will go out in the next release (ETA 2-3 days), after I've had a chance to see if anything else needs to be included in the release. Will tag you when the release is done.

**Comment by amodm** [comment]
@Azorlogh this is now released as [v0.7.0](https://github.com/amodm/webbrowser-rs/releases/tag/v0.7.0)

**Comment by Azorlogh** [comment]
Awesome, thanks!

**Pull request merged**